engine/apps/schedules/tests/test_on_call_schedule.py

```diff
@@ -246,14 +246,38 @@ def test_filter_events_ical_all_day(make_organization, make_user_for_organizatio
 
     events = schedule.final_events("UTC", start_date, days=2)
     expected_events = [
-        # all_day, users, start
-        (False, ["@Bernard Desruisseaux"], datetime.datetime(2021, 1, 26, 8, 0, tzinfo=pytz.UTC)),
-        (True, ["@Alex"], datetime.date(2021, 1, 27)),
-        (False, ["@Bob"], datetime.datetime(2021, 1, 27, 8, 0, tzinfo=pytz.UTC)),
+        # all_day, users, start, end
+        (
+            False,
+            ["@Bernard Desruisseaux"],
+            datetime.datetime(2021, 1, 26, 8, 0, tzinfo=pytz.UTC),
+            datetime.datetime(2021, 1, 26, 17, 0, tzinfo=pytz.UTC),
+        ),
+        (
+            True,
```

Comment:
Consider adding a test with all-day event spanning multiple days, because those can be tricky sometimes